public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -677,6 +677,7 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     meta?: DashboardMeta | DashboardWithAccessInfo<DashboardV2Spec>['metadata']
   ): void {
     this.serializer.initializeMapping(saveModel);
```

Comment:
```suggestion
    this.serializer.initializeElementMapping(saveModel);
```
Probably we could rename the element mapping. WDYT?